roles/redis/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+redis_docker_tag: 3.2-alpine
+redis_docker_state: started
+profile_docker_tag: latest
```

Comment:
Set this as latest since I'm assuming once we merge https://github.com/mozilla/fxa-profile-server/pull/254, the latest tag will be updated with the redis info.